internal/logtailer/tailer.go

```diff
@@ -213,6 +213,7 @@ func (t *logTailer) tailFile(seekTo *tail.SeekInfo) (err error) {
 		Location: seekTo,
 		ReOpen:   follow,
 		Follow:   follow,
+		Poll:     true,
```

Comment:
I don't think we want to do this in production. This is a logical race issue in the tests. The additional logs are being written to the file _before_ the tailer is up and read, so it doesn't notice the new lines, because there are no new lines.

Reply:
We could expose Poll to the test though, and use inotify in production with the caveat that the tests are testing the tailer.go file and not the underlying inotify library.